app/config.py

```diff
@@ -84,7 +84,7 @@ class Config(object):
     SQLALCHEMY_POOL_TIMEOUT = 30
     SQLALCHEMY_POOL_RECYCLE = 300
     SQLALCHEMY_STATEMENT_TIMEOUT = 1200
-    PAGE_SIZE = 50
+    PAGE_SIZE = 20
```

Comment:
does this have an associated admin-side PR?

Reply:
I was wondering about this too - I think it does, https://github.com/GSA/notifications-admin/pull/1882

Reply:
Yes, related to the pagination on UI side